doc/modules/calibration.rst

```diff
@@ -276,6 +276,54 @@ probabilities, the calibrated probabilities for each class
 are predicted separately. As those probabilities do not necessarily sum to
 one, a postprocessing is performed to normalize them.
 
+On the other hand, temperature scaling naturally supports multiclass
+predictions by applying the softmax function.
+
+Temperature Scaling
+^^^^^^^^^^^^^^^^^^^
+
+For a multi-class classification problem with :math:`n` classes, temperature scaling
+[9]_, `method="temperature"`, produces class probabilities by modifying the softmax
+function with a temperature parameter :math:`T`:
+
+.. math::
+       \mathrm{softmax}\left( \left[ \frac{z_1}{T},
+       \cdots, \frac{z_n}{T} \right] \right) \,,
+
+where :math:`\left[z_1, \cdots, z_n \right]` are the logits for each class produced
+by the calibrating estimator for a given sample. In terms of scikit-learn's API,
+these correspond to the outputs of either :term:`decision_function` or
+:term:`predict_proba`. Probabilities are converted to logits by first adding a small
+positive constant to avoid taking the logarithm of zero, and then applying the natural
+logarithm.
```

Comment:
```suggestion
:term:`predict_proba`. Probabilities are converted to logits by first adding a tiny
positive constant to avoid numerical issues with logarithm of zero, and then applying the natural
logarithm.
```